openslides/motions/access_permissions.py

```diff
@@ -138,6 +138,25 @@ def get_restricted_data(
         return data
 
 
+class ConstitutionParagraphAccessPermissions(BaseAccessPermissions):
```

Comment:
I prefer `statute` for German "Satzung" (https://dict.leo.org/englisch-deutsch/statute)

Reply:
ok